script.js

```diff
@@ -0,0 +1,68 @@
+let content;
+window.onload = function() {content = document.getElementById("content");}
+const doneTaskTextStyle = "line-through";
+
+function addEmptyTask(){
+    let task = createEmptyTask();
+    content.appendChild(task);
+}
+
+function createRawTask(){
+    // this function creates just the task object with the relevent event handlers, without inner elements
+    let newTask = document.createElement("div");
+    newTask.classList.add("task")
+    return newTask;
+}
+
+function addTaskInputBar(task){
+    // this function will add to the task the input bar section, and will return the input bar object.
+    let taskInput = document.createElement("input");
+    taskInput.classList.add("taskInputText");
+    taskInput.addEventListener("keydown",(event) => {
+        if (event.key == "Enter"){
+            taskInput.blur();
+        }
+    })
+    task.appendChild(taskInput);
+    return taskInput;
+}
+
+function addTaskDeleteButton(task){
+    let deleteButton  = document.createElement("button");
+    deleteButton.textContent = "X";
+    deleteButton.classList.add("deleteButton");
+    deleteButton.onclick = function(){deleteTask(task)};
+    task.appendChild(deleteButton);
+}
+
+function addTaskFinishedButton(task, taskInput){
+    let finishedButton  = document.createElement("button");
+    finishedButton.textContent = '\u{02713}';
+    finishedButton.classList.add("finishedButton");
+    finishedButton.onclick = function(){markDone(taskInput)};
+    task.appendChild(finishedButton);
+}
+
+function createEmptyTask(){
+    let newTask = createRawTask();
+    taskInput = addTaskInputBar(newTask)
+    addTaskDeleteButton(newTask)
+    addTaskFinishedButton(newTask, taskInput)
+    return newTask;
+}
+
+function deleteTask(task){
+    content.removeChild(task);
+    task.remove();
+}
+
+function markDone(finishedButton){
+    currentDecoration = finishedButton.style.textDecoration;
+    if (currentDecoration == doneTaskTextStyle){
```

Comment:
replace the `if else` with `cond ? iftrue : iffalse`

Reply:
not necessary anymore because i do not use the condition in this way anymore